docs/guide/cli.md

```diff
@@ -0,0 +1,126 @@
+# Command Line Interface
+
+## Dev server
+
+### `vite`
+
+Start Vite dev server in the current directory. Will enter the watch mode in development environment and run mode in CI automatically.
+
+#### Usage
+
+```bash
+vite [root]
+```
+
+#### Options
+
+| Options                  |                                                                 |
+| ------------------------ | --------------------------------------------------------------- |
+| `--host [host]`          | [string] specify hostname                                       |
```

Comment:
What if we follow [Vite](https://vitest.dev/guide/cli.html) here so we can start each description capitalized, and we also use inline code for the types following our config guide?
```suggestion
| `--host [host]`          | Specify hostname (`string`)                                      |
```

Reply:
Ok, I will rewrite it in that manner 👍